### PR TITLE
Add enr_message_board capability to kitchen demo

### PR DESCRIPTION
**Pull request by sktometometo**
Add enr_message_board feature to kitchen demo's trashcan capacity checking.

IMG_20230627_171251

With this PR, Fetch write a message about trashcan occupancy to enr_message_board.

For string - byte array conversion, see https://github.com/euslisp/jskeus/issues/635 for detail.

### CODE REVIEW

File: jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l [hunk 1]
```diff
@@ -654,4 +661,38 @@ Args:
     (send *ri* :go-pos-unsafe 0 0 -80) ;; face the front against the trash can
     success-move-to-trashcan-front))
 
+(defun padding-byte-array (byte-array target-length)
+  (concatenate cons byte-array (make-list (- target-length (length byte-array)) :initial-element 0)))
```
Comment (tkmtnt7000) [inline jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l:665]: Perhaps this function can be added to euslisp default code.(https://github.com/euslisp/euslisp or jskeus?)

File: jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l [hunk 2]
```diff
@@ -8,6 +8,7 @@
 (ros::load-ros-manifest "jsk_recognition_msgs")
 (ros::load-ros-manifest "power_msgs")
 (ros::load-ros-manifest "sensor_msgs")
+(ros::load-ros-manifest "esp_now_ros")
```
Comment (tkmtnt7000) [inline jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l:11]: Could you write something like the following?
I think that it is probably written to check for the existence of packages that have not been released like switchbot_ros with an if statement.
```suggestion
(if (ros::rospack-find "esp_now_ros")
    (ros::load-ros-manifest "esp_now_ros")
```

File: jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l (hunk 1 above)
Comment (sktometometo) [reply jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l:666]: Thank, I will add it if there is appropreate library.

**Review by tkmtnt7000 — DISMISSED** [review]
Thank you for updating. Nice feature.

**Comment by sktometometo** [comment]
@tkmtnt7000 Sorry, could you review this again?

**Review by tkmtnt7000 — APPROVED** [review]

**Pull request merged**